docs/policygenerator-reference.yaml

```diff
@@ -90,6 +90,12 @@ policies:
         # kustomization.yaml file. This path cannot be in a directory outside of the directory with
         # the kustomization.yaml file. Subdirectories within the directory with kustomization.yaml
         # file are allowed.
+        # Manifests format:
+        # 1) For policy-type manifests under manifests.path, which are already non-root policies (IamPolicy,
```

Comment:
Suggest to rephrase to:
Supported manifests:
1) Policy type manifests such as IamPolicy, CertificatePolicy and ConfigurationPolicy